src/main/java/org/mule/extension/db/internal/domain/connection/mysql/MySqlConnectionParameters.java

```diff
@@ -112,6 +113,11 @@ public String getUser() {
     return user;
   }
 
+  @Override
+  public java.util.Optional<TlsContextFactory> getTlsContextFactory() {
+    return java.util.Optional.empty();
```

Comment:
Same as Derby

Reply:
No idea, but if it does by implementing this method we would offer the feature.